shared/base.js

```diff
@@ -2,6 +2,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+import isReportableURL from "../shared/reportableUrl.js";
```

Comment:
The import path is wrong here, I think, since this file is already in `/shared/`.